ksql-engine/src/test/java/io/confluent/ksql/analyzer/QueryAnalyzerTest.java

```diff
@@ -40,11 +45,13 @@
 import io.confluent.ksql.parser.tree.QualifiedName;
 import io.confluent.ksql.parser.tree.QualifiedNameReference;
 import io.confluent.ksql.parser.tree.Query;
+import io.confluent.ksql.parser.tree.Sink;
 import io.confluent.ksql.util.KsqlException;
 import io.confluent.ksql.util.MetaStoreFixture;
 import io.confluent.ksql.util.Pair;
 import java.util.Arrays;
 import java.util.Collections;
+import java.util.Optional;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.rules.ExpectedException;
```

Comment:
Looks like this class doesn't have a test case for changing or preserving the result format based on the presence of VALUE_FORMAT in WITH. Since we're changing how we process the sink, it might make sense to add cases to make sure that all still works.

Reply:
done